packages/domain/lib/base/mappers/date_load_mapper.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:domain/base/mappers/base_mappers/mapper_base.dart';
+
+class IsTodayDateMapper extends Mapper<DateTime?, bool> {
+  @override
+  bool call(DateTime? params) {
+    if (params != null) {
+      final currentDate = DateTime.now();
+      return currentDate.year == params.year &&
+          currentDate.month == params.month &&
+          currentDate.day == params.day;
+    } else {
+      return false;
+    }
+  }
```

Comment:
up to you, but I would use the extension

Reply:
to be honest, this is not an extension) but it's better to be in the form of a mapper)

Reply:
I don't think it is a mapper. in Android, for ex it is implemented via a static method in the utils class. https://developer.android.com/reference/android/text/format/DateUtils#isToday(long). seems that we need smth like this